fedora/jellyfin.spec

```diff
@@ -50,18 +41,7 @@ Jellyfin is a free software media system that puts you in control of managing an
 
 
 %prep
-%autosetup -n %{name}-%{version} -b 0 -b 1
-web_build_dir="$(mktemp -d)"
-web_target="$PWD/MediaBrowser.WebDashboard/jellyfin-web"
-pushd ../jellyfin-web-%{version} || pushd ../jellyfin-web-master
-%if 0%{?fedora}
-nodejs-yarn install
-%else
-yarn install
-%endif
-mkdir -p ${web_target}
-mv dist/* ${web_target}/
-popd
+%autosetup -n jellyfin-server-%{version} -b 0
```

Comment:
```suggestion
%autosetup -n %{name}-%{version} -b 0
```

Reply:
I wanted to make this one explicit, since most of the other ones are (whether they override that name or not), but I can alter this if it's more standards-compliant!

Reply:
So yeah.  Reflecting on the ideas above about a metapackage and `jellyfin-server` subpackage, the 'Name:' in this specfile should still be `jellyfin` and the main package is the metapackage with `jellyfin-server` being a subpackage.

That's how this is all generally done, usually.

Reply:
How is this done when the two subpackages are completely separate though? Like, if both subpackages can have completely arbitrary versions and can be updated independently? That's the end goal. The metapackage is just to get the subpackages *installed* and might not ever even change versions, just grabbing whatever the latest subpackages are.